bin/github-linguist

```diff
@@ -22,10 +22,10 @@ end
 ARGV.shift
 breakdown = true if ARGV[0] == "--breakdown"
 json_breakdown = true if ARGV[0] == "--json"
-
```

Comment:
Please put this blank line back. It makes things clearer.

Reply:
You're right, it is re-added.